test.js

```diff
@@ -33,6 +33,13 @@ test('jams', t=>{
     t.ok(o)
     t.equal(o[0]['key'], 'val')
     t.equal(o[1], 'one')
+
+    try{
+        o = jams('{key{inner val}key{inner val}}')
```

Comment:
This should not throw, those are distinct objects and there are no duplicate keys

Reply:
Oh nvm I see “key” haha pls rename to key2

Reply:
isn't this an object with two inner jams that are identical?

Reply:
nvm missed your comment